### ISSUE TITLE
addRoute with array of methods

### ISSUE THREAD

**Issue by odan** [thephpleague/route#57]
Hello

I would like to pass the method as array like this:

```php
$router->addRoute(array('GET', 'POST'), '/', function (Request $request, Response $response) {
    // ...
});
```
Well this code is working but the documentation (and scrutinizer-ci) of the function addRoute
says that the datatype must be a string. 

```php
 * @param  string $method
```
But the internal call to the FastRoute addRoute function also accepts an array for $method.

```php
* @param string|string[] $httpMethod
```

https://github.com/nikic/FastRoute/blob/master/src/RouteCollector.php

Would it be possible to fix the DocBlock for the parameter $method to this?

```php
* @param string|string[] $method
```

**Comment by philipobenito** [comment]
@odan Would you like to provide a pull request? If so please do it against the 1.x branch.

**Comment by philipobenito** [comment]
Fixed #60
